bower.json

```diff
@@ -7,8 +7,8 @@
     "ember-inflector": "1.3.1",
     "ember-mocha": "~0.8.11",
     "Faker": "3.0.1",
-    "lodash": "~4.5.0",
-    "perfect-scrollbar": "~0.6.10",
+    "lodash": "^4.10.0",
+    "perfect-scrollbar": ">=0.6.7 <2.0.0",
```

Comment:
I'm not sure why we would trust perfect-scrollbar to be stable across a 1.x release - this should remain as a ~